keyboards/angel17/info.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
This info.json should be arranged like so, to match the `LAYOUT` macro in angel17.h (moved the `+` and Enter positions down one row):

```json
{
    "keyboard_name": "angel17",
    "url": "https://kakunpc.booth.pm/",
    "maintainer": "kakunpc",
    "width": 4,
    "height": 5,
    "layouts": {
        "LAYOUT_numpad_5x4": {
            "layout": [
                {"label":"Num Lock", "x":0, "y":0},
                {"label":"/", "x":1, "y":0},
                {"label":"*", "x":2, "y":0},
                {"label":"-", "x":3, "y":0},

                {"label":"7", "x":0, "y":1},
                {"label":"8", "x":1, "y":1},
                {"label":"9", "x":2, "y":1},

                {"label":"4", "x":0, "y":2},
                {"label":"5", "x":1, "y":2},
                {"label":"6", "x":2, "y":2},
                {"label":"+", "x":3, "y":1, "h":2},

                {"label":"1", "x":0, "y":3},
                {"label":"2", "x":1, "y":3},
                {"label":"3", "x":2, "y":3},

                {"label":"0", "x":0, "y":4, "w":2},
                {"label":".", "x":2, "y":4},
                {"label":"Enter", "x":3, "y":3, "h":2}
            ]
        }
    }
}
```

Reply:
Ah, yeah, I didn't even think about that. 

I think that once this is done, it should be good to go.